src/cls/tabular/cls_tabular_utils.cc

```diff
@@ -5864,7 +5885,7 @@ int transform_fb_to_arrow(const char* fb,
         }
 
         // Add entries for RID and Deleted vector
-        int num_cols = std::distance(sc.begin(), sc.end());
+        int num_cols = std::distance(query_schema.begin(), query_schema.end());
```

Comment:
I think you can use vector.size() here instead, not critical however.

Reply:
Done